src/Authentication/Authentication/Models/GraphOption.cs

```diff
@@ -0,0 +1,14 @@
+// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+using System.IO;
+
+namespace Microsoft.Graph.PowerShell.Authentication
+{
+    public class GraphOption : IGraphOption
```

Comment:
Keep the class internal to avoid misuse.
```suggestion
    internal class GraphOption : IGraphOption
```